Web/CSS2/styleAnimation.css

```diff
@@ -0,0 +1,27 @@
+.animation{
+    width: 100px;
+    height: 100px;
+	border-radius:120px 120px 120px 120px;
+    background-color: red;
+    position: relative;
+    -webkit-animation-name: showAnimation; 
```

Comment:
Use animation which takes multiple arguments instead of using name, duration and iteration-count differently.